tests/Formatting/Formatters/SpaceAfterBladeDirectivesTest.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+namespace Tests\Formatting\Formatters;
+
+use PHPUnit\Framework\TestCase;
+use Tighten\TLint\Formatters\SpaceAfterBladeDirectives;
+use Tighten\TLint\TFormat;
+
+class SpaceAfterBladeDirectivesTest extends TestCase
+{
+    /** @test */
+    public function it_adds_space_to_if_statement()
+    {
+        $file = <<<file
+@if(true)
+    This is true.
+@elseif(false)
+    This is false.
+@endforeach
+file;
+
+        $formatted = (new TFormat)->format(
+            new SpaceAfterBladeDirectives($file)
+        );
+
+        $correctlyFormatted = <<<file
+@if (true)
+    This is true.
+@elseif (false)
+    This is false.
+@endforeach
+file;
+
+        $this->assertEquals($correctlyFormatted, $formatted);
+    }
+
+    /** @test */
+    public function it_adds_space_to_unless_statement()
+    {
+        $file = <<<file
+@unless(true)
+    This isn't true.
+@endforeach
+file;
+
+        $formatted = (new TFormat)->format(
+            new SpaceAfterBladeDirectives($file)
+        );
+
+        $correctlyFormatted = <<<file
+@unless (true)
+    This isn't true.
+@endforeach
+file;
+
+        $this->assertEquals($correctlyFormatted, $formatted);
+    }
+
+    /** @test */
+    public function it_adds_space_to_for_statement()
+    {
+        $file = <<<file
+@for(\$i = 0; \$i < 10; \$i++)
+    The current value is {{ \$i }}
+@endfor
+file;
```

Comment:
Just a tip—if you put single quotes around the delimiter, the string behaves as if it's single-quoted, variables aren't interpolated, and you don't need to escape them.

```suggestion
        $file = <<<'file'
@for($i = 0; $i < 10; $i++)
    The current value is {{ $i }}
@endfor
file;
```

Reply:
Right! Thanks.